mail_footer_notified_partners/__openerp__.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#     This file is part of mail_footer_notified_partners,
+#     an Odoo module.
+#
+#     Copyright (c) 2015 ACSONE SA/NV (<http://acsone.eu>)
+#
+#     mail_footer_notified_partners is free software:
+#     you can redistribute it and/or modify it under the terms of the GNU
+#     Affero General Public License as published by the Free Software
+#     Foundation,either version 3 of the License, or (at your option) any
+#     later version.
+#
+#     mail_footer_notified_partners is distributed
+#     in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+#     even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+#     PURPOSE.  See the GNU Affero General Public License for more details.
+#
+#     You should have received a copy of the GNU Affero General Public License
+#     along with mail_footer_notified_partners.
+#     If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    'name': "Mail Footer Notified Partners",
+    'summary': """
+        This module adds into the footer the partner's name notified by this
+        email""",
+    'author': "ACSONE SA/NV",
```

Comment:
could you please add "Odoo Community Association (OCA)" to the author entry (by appending it to the string, separated by a comma)? This allows some visibility of OCA addons on apps.odoo.com.